src/MoBi.Presentation/MenusAndBars/ContextMenus/ContextMenuForModule.cs

```diff
@@ -43,10 +43,20 @@ public IContextMenu InitializeWith(ObjectBaseDTO dto, IPresenter presenter)
          _allMenuItems.Add(createSaveItemFor(module));
          _allMenuItems.Add(createRemoveItemFor(module));
          _allMenuItems.Add(createCloneMenuItem(module));
+
+         if(module.HasSnapshot)
+            _allMenuItems.Add(createRecreateMenuItem(module));
```

Comment:
rename `createRecreateMenuItem`. It's super hard to read
createNewFromSnapshotMEnuItem
Also naming is not consistant. all of them are ItemFor (witjoug Menu) it seems (or most of them at leasT)